src/modules/billing/jobs/create-charge.js

```diff
@@ -82,20 +88,25 @@ const handler = async job => {
     // Note: the await is needed to ensure any error is handled here
     return await updateBatchState(batch);
   } catch (err) {
-    // Always log and mark transaction as errored in DB
-    transactionsService.setErrorStatus(transactionId);
-    batchJob.logHandlingError(job, err);
-
-    // If not a client error, error the batch
-    if (!isClientError(err)) {
-      await batchService.setErrorStatus(batchId, BATCH_ERROR_CODE.failedToCreateCharge);
-      throw err;
+    const statusCode = get(err, 'statusCode');
+
+    // if the charge was created in the CM
+    if (statusCode === 409) {
+      batchJob.logHandlingError(job, err);
```

Comment:
May as well move:
```
 batchJob.logHandlingError(job, err);
```
to the top level catch block, then we don't need to repeat it below